cassandra-migration/src/main/resources/db/migration/cassandra/V1.24_cassandra.cql

```diff
@@ -1 +1,3 @@
+
+INSERT INTO system_settings(id, field, value) VALUES('userProfileConfig', 'userProfileConfig', '{ "userDataFields": ["firstName", "lastName", "profileSummary", "avatar", "countryCode", "dob", "email", "gender", "grade", "language", "location", "phone", "subject", "userName", "webPages", "jobProfile", "address", "education", "skills", "organisations", "badgeAssertions"], "eternalPublicFields": ["firstName", "lastName", "profileSummary"], "eternalPrivateFields": ["email", "phone"] }');
```

Comment:
We already ran 1.24 on Sunbird Dev. So, put this insert into V1.25_cassandra.cql.